lib/response.js

```diff
@@ -326,11 +326,14 @@ res.jsonp = function jsonp(obj) {
  */
 
 res.sendStatus = function sendStatus(statusCode) {
-  var body = statuses.message[statusCode] || String(statusCode)
-
+  // Emit deprecation warning for invalid status codes
+  if (typeof statusCode !== 'number') {
+    deprecate('Invalid status code: ' + String(statusCode) + '. Status code must be a number. This will throw an error in Express 6.');
```

Comment:
```suggestion
    deprecate('statusCode must be a valid number to res.sendStatus.');
```

I think keep it simple and only idea is just show deprecation warning.